src/xcode/ENA/ENA/Resources/Localization/bg.lproj/privacy-policy.html

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
-<html lang="bg">
+<html lang="en">
```

Comment:
NO!

Reply:
This is OK as the document is in the English language.